app/controllers/sessions_controller.rb

```diff
@@ -1,7 +1,7 @@
 class SessionsController < ApplicationController
   def create
     auth = request.env["omniauth.auth"]
-    account = ::Account.from_omniauth(auth)
+    account = BookingSyncEngine.setup_model.from_omniauth(auth, request.host)
```

Comment:
minor thing but as a verb it should be `set_up`

Reply:
Will change it into `account_model`. More readable IMO.